### DISCUSSION TITLE
bug: interface not saving "Allow user location" setting

### DISCUSSION THREAD

**Discussion by therohitdas** [open-webui/open-webui#4298] [General]
# Bug Report

## Description

**Bug Summary:** I am unable to save the ON value for Allow user location. I allowed the location when the browser asked for it. 

**Steps to Reproduce:**
1. Pull the latest version from docker.
2. Open the link, start a new chat.
3. Click on your profile and tap on setting.
4. Toggle Interface > Allow user location to ON and allow the browser prompt.
5. Close the pop-up.
6. Reopen it and the same setting is set to OFF.

**Expected Behavior:**
The setting should have retained the ON state

**Actual Behavior:**
The setting resets to OFF state. I checked the network request, the state is sent as FALSE. When I toggle to ON then OFF there is a console log: `{userLocation: false}` but no such log when I toggle OFF to ON.

## Environment

- **Open WebUI Version:** v0.3.10

- **Operating System:** macOS big sur
- **Browser (if applicable):** Arc Version 1.53.0 (52080)

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
First Models Console Log is when I toggle ON and SAVE.
The second where userLocation is False, I toggles OFF -> ON -> OFF
![Screenshot 2024-07-29 at 12 36 45 AM](https://github.com/user-attachments/assets/ad151cdd-631a-4f84-b4f6-991aed0bf536)

The Network request when I toggled OFF to ON and saved:
![Screenshot 2024-07-29 at 12 38 54 AM](https://github.com/user-attachments/assets/b9e5ae07-8b86-4e74-b8fe-a2c275cee16d)

## Installation Method

Docker method while also setting the volume path.

**Comment by tjbck** [comment]
Can't seem to reproduce on my end, please keep us updated if you find the culprit!

**Comment by sgrobpla** [comment]
I was able to reproduce it. I tried it in both Firefox and Chrome browsers. It appears that this issue only occurs in Firefox, while Chrome works correctly.

**Reply by hjbarraza** [reply]
It's happening to me in Safari
Starting asking for location, out of nowhere 
I just installed flux schnell function, but there seems nothing there that might be triggering it
